index.md

```diff
@@ -191,11 +192,6 @@ This is an overview of all Android versions and their corresponding identifiers
     <td><code>KITKAT</code></td>
     <td rowspan="3">2013</td>
   </tr>
-  <tr class="table-notes"><td colspan="3">
```

Comment:
I think we could keep the original one too, and change the wording to something like "Google Play services v12.34+ does not support below XX."

The older versions will continue working on older APIs, right?

Reply:
Alright, I can do that. I removed the old one since ICS was deprecated *before* Jellybean yet it didn't have a separate bullet. Should we put ICS as a bullet too?

Reply:
I've readded the Jellybean table for now. Let me know what you think.

Reply:
For consistency and helping friends who still have to maintain old versions, probably the ICS line would help too. Also if you know the "beyond" of Lollipop, please add that version too. These save a lot of research time if someone is looking for it.

Reply:
Alright, I added a bullet for ICS (I pray it looks right, you should test on your end, had to make sure everything looked right and the dates and everything *seems* to line up). For now, GMS hasn't bumped their minsdk to 21 as of yet (but they will very soon), so best I can do is put a more lenient statement by putting will instead of do so it sounds more like "in the near future". I didn't know what year to put the deprecation notice for ICS under so I just put it under 2012 to fit the rest of the table.

Reply:
Preview looks good to me.

2012 is the right year, because the year is meant for the API level, and this is a sub-note of the API level.

I like what you did with the "will + date", the statement will self-deprecate itself and someone will fix it later.